### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by ALI-WOSABY** [community/community#86649] [Other Feature Feedback, Questions, & Ideas]
### My GitHub account suddenly stopped and I was unable to access it

Question

### Body

I have an account on GitHub called Aliwosabi666 for my work with companies. The problem is that I did the verification using the authenticator app program. My problem is that I accidentally deleted the link through which the verification is done and the verification code was sent to me. Now I cannot access my account. I tried more than once and it was of no use, knowing that The email is correct: [programmeurweb.a@gmail.com](mailto:programmeurweb.a@gmail.com)

**Comment by mishmanners** [comment]
If you reach out to the GitHub support team, they should be able to help with this: https://docs.github.com/en/support/contacting-github-support

**Comment by ALI-WOSABY** [comment]
Please, how many days does it take for a response to the ticket from JibHub? I applied approximately three days ago and have not received any response from them until today, and my work has stopped

**Reply by mishmanners** [reply]
It depends on the nature of the ticket and if you have premium support or not.
